col.go

```diff
@@ -512,6 +518,61 @@ func (f *File) SetColWidth(sheet, startCol, endCol string, width float64) error
 	return err
 }
 
+// AutoFitColWidth provides a function to autofit columns according to
+// their text content
+// Note: this only works on the column with cells which not contains
+// formula cell and style with a number format.
+//
+// For example set column of column H on Sheet1:
+//
+// err = f.AutoFitColWidth("Sheet1", "H")
+//
+// Set style of columns C:F on Sheet1:
+//
+// err = f.AutoFitColWidth("Sheet1", "C:F", style)
+func (f *File) AutoFitColWidth(sheetName string, columns string) error {
```

Comment:
The same data type parameters can be simplified by one string data type declaration, this issue still not resolve, and the comments was not consistent with function signature.